support/src/traits.rs

```diff
@@ -112,3 +112,39 @@ pub trait RelayCallBuilder {
 	/// * debt: The weight limit used to process the call.
 	fn finalize_call_into_xcm_message(call: Vec<u8>, extra_fee: Self::Balance, weight: Weight) -> Xcm<()>;
 }
+
+pub trait EnabledFunctionality {
```

Comment:
I see this trait is implemented by the filter pallet, but where it is used?

Reply:
It was used in the staking & inflation pallet. In an old version those two could also be disabled.

Reply:
[a8797e8](https://github.com/KILTprotocol/kilt-node/pull/397/commits/a8797e86e907f8dc20010f49ba83386ceda08bbd)